### PR TITLE
[11.0][MIG] web_searchbar_full_width

### PR DESCRIPTION
**Pull request by carlos-domatix**

### CODE REVIEW

File: web_searchbar_full_width/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yajo) [inline web_searchbar_full_width/__init__.py]: Remove all of this.

File: web_searchbar_full_width/README.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    :alt: Searchbar over full screen width
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/web/issues/new?body=module:%20web_searchbar_full_width%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (yajo) [inline web_searchbar_full_width/README.rst]: Update template

File: web_searchbar_full_width/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (yajo) [inline web_searchbar_full_width/__manifest__.py]: Use new, shorter, headers please.

**Review by yajo — APPROVED** [review]
Migration looks quite good, just some little suggestions to upgrade code to new standards.

File: web_searchbar_full_width/__init__.py [hunk 4]
```diff
@@ -1,19 +1,2 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (yajo) [inline web_searchbar_full_width/__init__.py]: I think you can completely ~~drop~~ empty this file. It's copyrighting 0 LOC.

In any case, it's AGPL

File: web_searchbar_full_width/__manifest__.py [hunk 5]
```diff
@@ -1,22 +1,6 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2015 Therp BV <http://therp.nl>.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
```
Comment (yajo) [inline web_searchbar_full_width/__manifest__.py]: AGPL

**Review by yajo — CHANGES_REQUESTED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
LGTM. Same remarks as Jairo

**Review by simahawk — CHANGES_REQUESTED** [review]
wrong button :)

**Comment by lfreeke** [comment]
:+1: (test on runbot)

**Comment by pedrobaeza** [comment]
As @Yajo's comments have been honoured, I merge.

**Pull request merged**